lib/player.rb

```diff
@@ -0,0 +1,24 @@
+class Player
+  attr_accessor :player_move
+  attr_reader :name, :icon
+  def initialize(name, icon)
+    @name = name
+    @icon = icon
+    @player_move = []
+  end
+
+
+  def move(board)
```

Comment:
Not required but good for further improvement:

It would be better to separate user interface related logic from the class. For instance here you are getting the user input from the Player class. The better way to do this is have a module that will have all the UI related code and just use the relevant method in that module here. You would want your classes to be reusable and loosely coupled with other objects as much as possible. Reason for this is that we need to learn to build programs that are extendable and not requiring lots of modifications in the current code to extend the features. This is part of why OOP was developed. For instance, say, later on, if this was a real app in a company, you were assigned to use a library/gem that will handle all the user inputs for you. Then in that case you will need to revise this Player class in order to accomodate the use of that gem (you will then have to delete lines 12 - 21 + other modifications). The Player class should be versatile and independent of how the user chooses the input (whether by mouse click or typing in the terminal etc).